### PR TITLE
Update screening and eligibility requirements for `Official tournament support`

### PR DESCRIPTION
**Pull request by DRCallaghan**
## Changes
  - Further clarified the expected use of the tournament report form.
  - Removed second iteration or experienced staff team requirements from eligibility.
  - Added adherence to expectations of officially supported tournaments to eligibility requirements.
  - Removed mentions of experienced staff and second iteration requirements from FAQ and Notes.
  - Added paragraph in Tournament Screening section regarding team tournament submission with large amount of players for screening.

## Eligibility Requirement Changes Reasoning
- Currently and over the past 5 years, many new tournaments have used one experienced administrator as a way to badge a first iteration.
- These experienced administrators usually do very little or no work in running the tournament. Effectively, these tournaments are run without the support of an experienced staff or administration.
- In spite of that, due to having an experienced administrator on the staff team, these tournaments still get badged.
- Many of these tournaments would likely be unbadged with further encouragement of use of the official report function.
- Additionally, there are many tournaments with fully experienced staff teams that would not be able to be badged if the second iteration requirement were kept, e.g. the Perennial or the Roundtable.
- Judging tournaments based on their execution ensures a fair playing ground for all tournaments regarding eligibility for official support.

## Screening Changes Reasoning
- Currently, many large country-based tournaments send screening lists of thousands of players to staff. 
- The screening process is only partially automated, meaning these lists take hours to be parsed.
- The vast majority of the players on these screening lists often do not play in the tournament at all; only several hundred out of 2.700+ will be able to play in the current iteration of the 5 Digit World Cup, for example.
- Hosts of this style of tournament only realistically need to send in the players actually on each country's team for this style of tournament for screening, along with a few substitutes to ensure that even if players on each country's team do get screened out, a second round of screening is not needed.
- The wording on this change intentionally does not mention country-based tournaments outside of its example due to the possibility of there being other tournament formats this could apply to.
- This change will expedite screening time for the largest of tournaments following a format that allows for this limited screening by many hours and will help streamline the screening process overall.

## Self-check

- [x ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/Official_support/de.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ tags:
   - Abzeichen
 outdated_translation: true
 outdated_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
+
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/de.md:9]:
```suggestion
```

File: wiki/Tournaments/Official_support/fr.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ tags:
   - badged
 outdated_translation: true
 outdated_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
+
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/fr.md:8]:
```suggestion
```

**Review by Walavouchey — COMMENTED** [review]
in the future please always [start of with a clean branch](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Line 14 needs to have its date updated (can't add a review, apply manually):

```markdown
This article was last updated on February 2, 2023. For any queries and clarifications, please consult the `#tournaments` channel on the [osu!dev Discord server](https://discord.com/invite/ppy) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

File: wiki/Tournaments/Official_support/en.md [hunk 3]
```diff
@@ -121,6 +121,8 @@ User4,Team2,1234567
 
 Once screening concludes, the account support team will provide a list of any users who failed screening and are not considered eligible for tournament play, without providing specific reasoning. Individual users who are unhappy with their screening outcome should be told to consult [accounts@ppy.sh](mailto:accounts@ppy.sh) via email.
 
+Only players who are participating in the tournament or would be likely substitutes for a team made during the course of the tournament should be sent for screening. For example, in a country-based tournament using a tryouts system, organizers are encouraged to send only the players determined to be participating after the tryouts process concludes, along with several substitutes for each team in the event one or more of the players on a given country's team get screened. For clarifications on whether a tournament should send all registrations or only relevant registrations, please contact a member of the [Tournament Committee](/wiki/People/Tournament_Committee).
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:124]: part of the first sentence i believe is redundant. i'm not sure if there's an intentional distinction to make here (or whether it makes sense).

last sentence sounds a bit baffling. if i'm off the mark please explain.

```suggestion
Only players who are participating in the tournament or would be likely substitutes for a team should be sent for screening. For example, in a country-based tournament using a tryouts system, organisers are encouraged to send only the players determined to be participating after the tryouts conclude, along with several substitutes for each team in the event they need to participate. For clarifications on which registrants are relevant for screening for a particular tournament, contact the [Tournament Committee](/wiki/People/Tournament_Committee).
```

**Review by LeoFLT — DISMISSED** [review]

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Official_support/en.md (hunk 3 above)
Comment (DRCallaghan) [reply wiki/Tournaments/Official_support/en.md:124]: The first sentence is not redundant. This change is specifically for tournaments where completed rosters are not submitted with registrations. Tournament Committee does not want this change to apply to all team-based tournaments, only to those where rosters are formed and submitted for screening during the course of the tournament.

All other changes are good.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**